apps/typegpu-docs/src/utils/examples/exampleContent.ts

```diff
@@ -1,38 +1,64 @@
-import {
-  entries,
-  filter,
-  fromEntries,
-  groupBy,
-  isNonNull,
-  map,
-  pipe,
-} from 'remeda';
-import { parseExampleCode } from './parseExampleCode';
-import type { Example } from './types';
-
-const rawExamples: Record<string, string> = import.meta.glob(
-  '../../content/examples/**/*.ts',
-  {
+import { entries, filter, fromEntries, groupBy, map, pipe } from 'remeda';
+import type { Example, ExampleMetadata } from './types';
+
+function pathToExampleKey<T>(record: Record<string, T>): Record<string, T> {
+  return pipe(
+    record,
+    entries(),
+    map(
+      ([path, value]) =>
+        [
+          pipe(
+            path,
+            (path) => path.replace(/^..\/..\/content\/examples\//, ''), // remove parent folder
+            (path) => path.replace(/\/[^\/]*$/, ''),
+            (path) => path.replace(/\//, '--'),
```

Comment:
Tiny nit, a bit more docs
```suggestion
            (path) => path.replace(/^..\/..\/content\/examples\//, ''), // remove parent folder
            (path) => path.replace(/\/[^\/]*$/, ''), // removing leaf filenames (e.g. meta.json, index.ts)
            (path) => path.replace(/\//, '--'), // replacing path separators with '--'
```